constants.ts

```diff
@@ -0,0 +1,36 @@
+
+export const L16_RPC_URL = 'https://rpc.l16.lukso.network';
+
+export const L14_RPC_URL = 'https://rpc.l14.lukso.network';
```

Comment:
I think we just want to deploy this on l16? Do we need the l14 RPC url?

Reply:
This could also be configured as an env var if we want to deploy to both

Reply:
yep will remove it. just was facing many issues with l16 so I worked for a little bit on L14